### ISSUE TITLE
gnome-shell: Search results not aligned in overview

### ISSUE THREAD

**Issue by drakkar1969** [jnsh/arc-theme#92]
gnome-shell 3.38.1, latest git version of Arc theme

Search results in the overview have different vertical alignments, resulting in different sizes of highlight boxes. This only seems to affect system actions (e.g. restart/power off):

![search-alignment](https://user-images.githubusercontent.com/18715287/95674524-18e96d00-0bb1-11eb-8a0c-21f7f4d77412.png)

I have narrowed down issue to the `@extend %icon_tile` code in this block:

```scss
.app-well-app,
.app-well-app.app-folder,
.grid-search-result {

  .overview-icon {
    @extend %icon_tile;
  }
  ...
}
```

The issue can be solved by deleting the `padding` property from the `%icon-tile` snippet:

```scss
%icon_tile {
  color: $_shell_fg_color;
  border-radius: 2px;
  padding: 6px;
  border: 1px solid transparent;
  transition-duration: 0ms;
  text-align: center;
}
```

However, I don't think simply deleting the `padding`  property is the way to go, since it is present in the upstream code.

**Comment by jnsh** [comment]
Thanks, this was rather tricky, but I managed to figure out the culprit with looking glass.

Apparently this used to be the case also for the default shell theme, and got fixed (intentionally or not) during the 3.36 refactor. I should go trough the Arc shell theme and refactor it against the current upstream to fix other similarly outdated bits, but there's some major GTK work that I have to get done first.
